rpm/fs/etc/init.d/mad

```diff
@@ -0,0 +1,167 @@
+#!/bin/sh
```

Comment:
I would hope that it's a reasonable assumption to have sh available on the system. Do rpms usually declare this dependency?